compose.yaml

```diff
@@ -18,6 +18,21 @@ services:
       context: ./frontend/
     ports:
       - 5001:8080
+  mongo:
+    build:
```

Comment:
```suggestion
  mongo:
    env_file: "mongodb.env"
    build:
```
```env
# mongodb.env example
MONGO_INITDB_ROOT_USERNAME=mongoadmin
MONGO_INITDB_ROOT_PASSWORD=mongopass
MONGO_PORT=5002
```

we can start using env for making this less prone to mistakes, the example .env file can look like this so everyone knows they have to add the values for local development

```env
# For local development add the missing config
MONGO_INITDB_ROOT_USERNAME=
MONGO_INITDB_ROOT_PASSWORD=
MONGO_PORT=
```

Reply:
env_file is a nice idea yeah. I'll use a general one to use as config file with all ports as you also suggest.

Reply:
I'll keep the environment variables as arguments in the Dockerfiles as container "default" values.

According to official docs, env file takes precedence and overwrites those if any dev changes those.
https://docs.docker.com/compose/environment-variables/envvars-precedence/